whirlpool/dryer.py

```diff
@@ -0,0 +1,288 @@
+import logging
+from enum import Enum
+
+from .appliance import Appliance
+
+LOGGER = logging.getLogger(__name__)
+
+# Machine State
+ATTR_MACHINE_STATE = "Cavity_CycleStatusMachineState"
+
+ATTRVAL_MACHINE_STATE_STANDBY = "0"
+ATTRVAL_MACHINE_STATE_SETTING = "1"
+ATTRVAL_MACHINE_STATE_DELAYCOUNTDOWNMODE = "2"
+ATTRVAL_MACHINE_STATE_DELAYPAUSE = "3"
+ATTRVAL_MACHINE_STATE_SMARTDELAY = "4"
+ATTRVAL_MACHINE_STATE_SMARTGRIDPAUSE = "5"
+ATTRVAL_MACHINE_STATE_PAUSE = "6"
+ATTRVAL_MACHINE_STATE_RUNNINGMAINCYCLE = "7"
+ATTRVAL_MACHINE_STATE_RUNNINGPOSTCYCLE = "8"
+ATTRVAL_MACHINE_STATE_EXCEPTIONS = "9"
+ATTRVAL_MACHINE_STATE_COMPLETE = "10"
+ATTRVAL_MACHINE_STATE_POWERFAILURE = "11"
+ATTRVAL_MACHINE_STATE_SERVICEDIAGNOSTIC = "12"
+ATTRVAL_MACHINE_STATE_FACTORYDIAGNOSTIC = "13"
+ATTRVAL_MACHINE_STATE_LIFETEST = "14"
+ATTRVAL_MACHINE_STATE_CUSTOMERFOCUSMODE = "15"
+ATTRVAL_MACHINE_STATE_DEMOMODE = "16"
+ATTRVAL_MACHINE_STATE_HARDSTOPORERROR = "17"
+ATTRVAL_MACHINE_STATE_SYSTEMINIT = "18"
+ATTRVAL_MACHINE_STATE_MYSTERY = "19" # Observed on "Maytag MGD6230HW3"
+
+# Cavity
+ATTR_OP_STATUS_DOOROPEN = "Cavity_OpStatusDoorOpen"
+ATTR_TIME_STATUS_ESTTIMEREMAINING = "Cavity_TimeStatusEstTimeRemaining"
```

Comment:
```suggestion
ATTR_TIME_REMAINING = "Cavity_TimeStatusEstTimeRemaining"
```
we can simplify the naming to make them easier to read/understand